### ISSUE TITLE
Card with no name not working

### ISSUE THREAD

**Issue by edautz** [dgthomson/nodered-shellrecharge#28]
I using this subflow now few years without a problem. Using version sr-2-50.

Although this week I stumble across a problem.

I switched my leasecar with a privat one and starting to use my privat card for my newmotion advanced charging station.

This inflicts a problem. I think it is boiled down due the fact that the new card doesn't has a name in the shell recharge environement.
Adding a name afterwards in the app and the webconsole doesn't help. The name doesn't appear in the the setup information.

This results in an error that the rfid of this card is not recognised when trying to start charging . Other functions like update the state works fine.

Manually starting of through the shell app works fine.
  
Setup information of card 3 and 4. Cleared some information for security reasons.

![setup](https://github.com/user-attachments/assets/5f2b7cb8-5290-44d3-b124-6405f2e67f3f)

Old card with index 3 has a name. New card with index 4 no name.

![Working card 3](https://github.com/user-attachments/assets/05ecc8e8-0b2e-4f43-bce2-b088241c17f6)

Usding start with card 3 although this card is blocked by the provider.

![Error card 4](https://github.com/user-attachments/assets/860ce505-109a-45ea-872a-7cae6af339eb)

Error message when using start with card with index 4.

Any help appreciated.

**Comment by dgthomson** [comment]
Not sure. I've uploaded a version edautz.json which has some extra debugging nodes- can you try it, and copy the debugging messages here?

**Comment by edautz** [comment]
Thnxs for your fast respons and support. Very appreciated.

I imported this new subflow, bu this leads to an authentication problem. Tested setup and start.

![Test1](https://github.com/user-attachments/assets/27de12ee-5370-4d2d-bafa-46b9a7c16c24)

**Comment by dgthomson** [comment]
Sorry. Try edautz2.json

**Comment by edautz** [comment]
This version works as a charm. In setup I see the new card with name. And charging start/stop is working.
Are you interrested in the debug output?

**Comment by dgthomson** [comment]
I didn't actually change anything. Glad it's working though.

**Comment by edautz** [comment]
Very strange. I see that the new node produced the new card with name, while the old node still doesn't.
Output new node:

![Setup new](https://github.com/user-attachments/assets/041acf92-fc53-4415-a034-64326ceea693)

But this works for me. Thanks for your swift support.

**Comment by dgthomson** [comment]
There was probably some cached data somewhere. If you inject...

`msg.payload = { "action": "flush" }`

...into the old node it'll probably start working.

**Comment by edautz** [comment]
Tested that and that was the case. Thank you again.
